WebDriverAgentTests/UnitTests/FBElementCacheTests.m

```diff
@@ -53,4 +53,65 @@ - (void)testResolvingFetchedElement
   XCTAssertTrue(element.didResolve);
 }
 
+- (void)testLinearCacheExpulsion
+{
+  NSMutableArray *elements = [NSMutableArray arrayWithCapacity:1050];
```

Comment:
Please use constants instead of magic numbers

Reply:
Makes sense, updated the tests.